### PR TITLE
[Student][MBL-13637] Set a11y label for account preferences toggle

### PR DESCRIPTION
**Pull request by JordanMarshall**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1014446&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/student/src/main/res/layout/fragment_account_preferences.xml [hunk 1]
```diff
@@ -101,8 +101,9 @@
                     android:layout_alignParentStart="true"
                     android:layout_centerVertical="true"
                     android:layout_toStartOf="@+id/mondayToggle"
+                    android:labelFor="@id/mondayToggle"
```
Comment (joehoag) [inline apps/student/src/main/res/layout/fragment_account_preferences.xml]: So, "Calendar starts on Monday" is *always* the label for mondayToggle?  Shouldn't the label depend on the setting of the toggle?

Comment (JordanMarshall) [reply apps/student/src/main/res/layout/fragment_account_preferences.xml]: TalkBack also reads the state of the switch as well as the label

Comment (joehoag) [reply apps/student/src/main/res/layout/fragment_account_preferences.xml]: 👍

**Pull request merged**